packages/components/src/internal/components/files/FileAttachmentContainer.tsx

```diff
@@ -117,7 +118,7 @@ export class FileAttachmentContainer extends React.Component<
                     invalidNames = invalidNames.add(file.name);
                 }
             }
-            if (sizeLimits) {
+            if (sizeLimits && !allowOversize) {
```

Comment:
Can we just not provide sizeLimits if we want to ignore them (probably based on the experimental feature flag for now)?

Reply:
good point, done.